src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js

```diff
@@ -171,6 +171,18 @@
                 }
             }));
 
+            evts.push(eventsService.on("rte.file.uploading", function(){
+                notificationsService.info("Saving Disabled", "The node cannot be saved until the image/s has finished uploading")
```

Comment:
This is actually not necessary and i think might get annoying, turns out that TinyMCE has it's own image upload progress, but this can only be seen when running on a slower connection, if you throttle to slow 3g, you'll see it, example:

![image](https://user-images.githubusercontent.com/1742685/65216439-37874c80-daf4-11e9-85d0-c95c5d6f7f7b.png)